runtime/ftplugin/vim.vim

```diff
@@ -47,6 +47,7 @@ setlocal fo-=t fo+=croql
 " keyword character.  E.g., for netrw#Nread().
 setlocal isk+=#
 
+" Prefer Vim help instead of manpages.
 " Use :help to lookup the keyword under the cursor with K.
 setlocal keywordprg=:help
```

Comment:
```suggestion
```
Please remove this and leave the other place to match the upstream ftplugin (helps prevent this cropping up again).